pkg/backends/newrelic/newrelic.go

```diff
@@ -486,6 +539,22 @@ func newInfraPayload(data interface{}) NewRelicInfraPayload {
 	}
 }
 
+func (n *Client) setTags(tags gostatsd.Tags, data *map[string]interface{}) {
```

Comment:
You don't need to pass a pointer to a map since a map object itself is already a reference to a memory block.